src/modules/shared/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -6,49 +6,68 @@ import { ReactComponent as HomeIcon }
   from '../../../static/icons/home_icon.svg';
 import { ReactComponent as ArrowRightIcon }
   from '../../../static/icons/arrow-right_icon.svg';
-
 import styles from './Breadcrumbs.module.scss';
 
 export const Breadcrumbs: React.FC = () => {
   const location = useLocation();
   const { pathname } = location;
   const segments = pathname.split('/');
-  const currentPage = (segments[segments.length - 1]);
+  const currentPage = segments[segments.length - 1];
   const { isDarkTheme } = useAppSelector((state) => state.theme);
   const { origin } = window.location;
 
   let url = '';
-  const breadcrumbLinks = segments.map((segment) => {
+  const breadcrumbLinks = segments.map((segment, index) => {
     url += `${segment}`;
 
+    const isLastSegment = index === segments.length - 1;
+
     return (
       <React.Fragment key={segment}>
-        <Link to={`${origin}/${url}`}>
-          {segment === ''
-            ? (
-              <HomeIcon
-                color={isDarkTheme ? '#f1f2f9' : '#0f0f11'}
+        {isLastSegment ? (
+          <div className={styles.link}>
+            {segment === '' ? (
+              <HomeIcon color={isDarkTheme ? '#f1f2f9' : '#0f0f11'} />
+            ) : (
```

Comment:
A lot of ternary opetaros looks a bit confusing.  Think if it can be done a bit more nicely. At least put '?' and ':' on the next lines